packages/firebase_data_connect/firebase_data_connect/lib/src/network/grpc_transport.dart

```diff
@@ -0,0 +1,137 @@
+// Copyright 2024, the Chromium project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+part of firebase_data_connect_grpc;
+
+/// Transport used for Android/iOS. Uses a GRPC transport instead of REST.
+class GRPCTransport implements DataConnectTransport {
+  /// GRPCTransport creates a new channel
+  GRPCTransport(this.transportOptions, this.options, this.auth, this.appCheck) {
+    bool isSecure =
+        transportOptions.isSecure == null || transportOptions.isSecure == true;
+    channel = ClientChannel(transportOptions.host,
+        port: transportOptions.port ?? 443,
+        options: ChannelOptions(
+            credentials: (isSecure
+                ? const ChannelCredentials.secure()
+                : const ChannelCredentials.insecure())));
+    stub = ConnectorServiceClient(channel);
+    name =
+        'projects/${options.projectId}/locations/${options.location}/services/${options.serviceId}/connectors/${options.connector}';
+  }
+
+  /// FirebaseAuth
+  FirebaseAuth? auth;
```

Comment:
```suggestion
  @override
  FirebaseAuth? auth;
```

Reply:
If I leave the override out, I get a lint error saying that I need to add the `@override` annotation